java/src/jmri/jmrit/beantable/AbstractLogixNGTableAction.java

```diff
@@ -269,7 +269,7 @@ protected void configDeleteColumn(JTable table) {
                 editCombo.addItem(Bundle.getMessage("ButtonSelect"));  // NOI18N
                 editCombo.addItem(Bundle.getMessage("ButtonEdit"));  // NOI18N
                 editCombo.addItem(Bundle.getMessage("BrowserButton"));  // NOI18N
-                editCombo.addItem(Bundle.getMessage("ButtonCopy"));  // NOI18N
+                if (isCopyBeanSupported()) editCombo.addItem(Bundle.getMessage("ButtonCopy"));  // NOI18N
```

Comment:
Just a question, not a request for change:  Would it be better to add the button always and disable it if not supported?

Reply:
It's a JComboBox. Is it possible to disable an item in a combo box?

Reply:
I thought there was, but now that I'm looking for it, I don't see a way to do it.  Sorry for the confusion.

Reply:
No problem!